### DISCUSSION TITLE
Welcome to obsidian-excalidraw-plugin Discussions!

### DISCUSSION THREAD

**Discussion by zsviczian** [zsviczian/obsidian-excalidraw-plugin#137] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ixoulnehc-sudo** [comment]
How can I use Arial text other than default?

**Comment by zsviczian** [comment]
@ixoulnehc-sudo, I recommend asking these questions from the NotebookLM agent. You'll get quality instant responses: https://notebooklm.google.com/notebook/42d76a2f-c11d-4002-9286-1683c43d0ab0

**Comment by transparentWind** [comment]
I wish to add a function to draw a circular arc, which is rounder than a curve.

**Comment by zsviczian** [comment]
@transparentWind did you try to prompt AI to write the script for you? 
https://youtu.be/6BjhUyfS4iM

Alternatively you could point Claude code to  https://github.com/zsviczian/ea-script-template and ask it for the script. It will know what to do.

**Comment by AldarArmaev** [comment]
Hello! I recently discovered Excalidraw and really like it. I wanted to ask if there is a way to add a feature that converts handwritten input into LaTeX?

**Comment by zsviczian** [comment]
@AldarArmaev, I am pretty confident that with a few hours of focused prompting (maybe faster) Codex or Cloude Code can create the Excalidraw script for you that will achieve this.

You could prompt Claude or Codex something like this:

```
Using this [template repository](https://github.com/zsviczian/ea-script-template), create an Excalidraw script for me that converts handwritten LaTeX to LaTeX elements in the scene.
```
